tests/pytorch/fused_attn/test_fused_attn.py

```diff
@@ -381,6 +393,21 @@ def test_dpa_sliding_window(dtype, model_configs, model):
     """Test DotProductAttention module with sliding window attention"""
     test_dot_product_attention(dtype, model_configs, model, False, True, None, True)
 
+model_configs_alibi_slopes = {
+    #     test:             b,  h, hg,   d,   sq,  skv,   p,      mask,    bias, alibi_type
+    "alibi_1_0": ModelConfig(4, 16, 16,  64,  128,  128, 0.0, "causal", "alibi", alibi_type="vanilla"),
+    "alibi_1_1": ModelConfig(2, 16, 16,  64,  128,  256, 0.0, "causal", "alibi", alibi_type="vanilla"),
+    "alibi_2_0": ModelConfig(4, 24, 24, 128, 2048, 2048, 0.0, "causal", "alibi", alibi_type= "custom"),
+    "alibi_2_1": ModelConfig(2, 24, 24, 128, 2048, 4096, 0.0, "causal", "alibi", alibi_type= "custom"),
```

Comment:
Can we get away with smaller configurations? Full correctness testing should be handled in cuDNN and Flash Attention. I propose b=1, h=1, hg=1, d=32, sq={128,1024}, skv={256,2048}.

Reply:
I reduced `b` to 2 and 1, and `sq` to {128, 1024}, `skv` to {256, 2048}. I kept `h`/`hg` and `d` though. I think it's better to test some real-life cases here than not. ALiBi slopes is a different path from some of the other tests.